commands/features/disable.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+let cli = require('heroku-cli-util');
+let co  = require('co');
+
+function* run (context, heroku) {
+  let app     = context.app;
+  let feature = context.args.feature;
+
+  yield cli.action(`Disabling ${cli.color.green(feature)} for ${cli.color.cyan(app)}`, co(function* () {
```

Comment:
@dickeyxxx never mind, I am looking at the ruby code for features listing and it makes more sense now.